src/i18n/en/translation.en.json

```diff
@@ -568,12 +568,8 @@
     "members-view": {
       "title": "Active community members"
     },
-    "discussion-category-selector": {
-      "ALL": "SHOW ALL",
-      "GENERAL": "GENERAL",
-      "IDEAS": "IDEAS",
-      "QUESTIONS": "QUESTIONS",
-      "SHARING": "SHARING"
+    "category-selector": {
+      "all": "SHOW ALL"
```

Comment:
I'd rather put normal-case strings here and capitalize in CSS since capitalization looks like styling and not content.